clubs_guides/urls.py

```diff
@@ -1,3 +1,18 @@
 from django.conf.urls import url
 
-# Create you url mappings here.
+from clubs_guides.views import (
+    ClubsGuidesListView,
+    ClubsGuideView,
+    CategoryListView,
+)
+
+
+urlpatterns = [
```

Comment:
> It might even be worth explicitly naming the routes, tied to the code that services them, instead of relying on regex

I'm not sure I follow. The Django convention for url mapping is to use regex with named groups (like `pk`) to assign routes.

I will add the comments though!